forc-pkg/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "forc-pkg"
+version = "0.6.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+homepage = "https://fuel.network/"
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/sway"
+description = "Fuel Orchestrator."
+
+[dependencies]
+anyhow = "1"
+forc-util = { version = "0.6", path = "../forc-util" }
+git2 = "0.14"
+petgraph = { version = "0.6.0", features = ["serde-1"] }
```

Comment:
While we're here, could we make these non-Fuel dependencies only major.minor instead of major.minor.patch? Could be left to a future PR.

Reply:
Good catch for the petgraph dep, I think it makes sense here!

That said, I've noticed this recommended in other fuel repos even when the specified patch version is non-zero. Is this mostly for manifest style reasons? Or because we always want cargo to be able to pick from any patch version (even prior to the one released at the time the dep was introduced)?

Worth noting as it's not always clear when getting the hang of cargo: Running `cargo update` for a patch-specified version will still update the lock to newer patch versions that follow the specified patch release. Specifying the patch will only limit cargo from selecting older patch versions, which can be handy if the version specified was the known working version at the time of dev/testing. It can of course also be handy if you need to make sure it has a particular patch, or you just want to signal what version was the latest when the dependency was added.